Src/AutoFixture.xUnit.net.UnitTest/CompositeDataAttributeInsufficientDataTest.cs

```diff
@@ -74,17 +74,7 @@ public IEnumerator<object[]> GetEnumerator()
             yield return CreateTestCase(
                 data: new[]
                 {
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] {  new object[] { 1, 2    },  new object[] {  3,  4     }, new object[] { 5, 6 } }),

```

Comment:
These both test whether the `CompositeDataAttribute` throws an `InvalidOperationException` if the first attribute provides more data sets than the rest. This used to be expected behaviour, but now the exception shouldn't be thrown - instead, the attribute providing the fewest number of data sets determines the number of data sets that are combined + provided to the theory. So I have moved these tests to the `CompositeDataAttributeSufficientDataTest` file, as they now contain sufficient data.

As a side note the `ComposeTwoPropertiesAttribute` described above *should* throw an exception: Neither `Property1` nor `Property2` provide a full set of data, so the third parameter `z` can never be populated